examples/print_to_pdf.rs

```diff
@@ -0,0 +1,36 @@
+use std::{env, fs};
+use headless_chrome::{protocol::page::PrintToPdfOptions, Browser};
+
+/// Run the example as follows:
+/// ./print_to_pdf /rust-headless-chrome/tests/pdfassets/index.html
+fn main() -> Result<(), failure::Error> {
+    // Create a headless browser, navigate to index.html file, wait for the page
+    // to render completely, render a PDF of the page and save it to the filesystem.
+
+    // webSocketDebuggerUrl is obtained by GET http request to 
+    // http://{chromeservicehost}{port}/json/version
+    // and pass an empty-value host param: {host: ""} in header 
+    let file_path = format!("file://{}", 
+                            env::args().nth(1).expect("Must provide path/to/file/index.html"));
+    let debug_ws_url = "ws://127.0.0.1:9222/devtools/browser/14804b82-0392-43be-b20f-d75678460e43";
```

Comment:
Is this `UUID` generic?

Reply:
@lukaslueg that uuid is unique to this example and has to be replaced and recompiled in order for the example to work.  I've updated my PR with this part externalized, requiring it as a parameter